src/Pages/CollectionPage/CollectionPage.js

```diff
@@ -1,4 +1,41 @@
+import CollectionListItem from '../../Components/CollectionListItem/ListItem.js';
+import bro from '../../assets/images/bro.png';
+import { collection } from '../../data.js';
+import CollectionCard from '../../Components/CollectionCard/CollectionCard.js';
+import FloatingBtn from '../../Components/floatingBtn/floatingBtn.js';
+
 function CollectionPage() {
-  return <div>Collection Page</div>;
+  return (
+    <div className="flex h-full bg-lightGrey">
+      <div className="h-5/6">
+        <div className="bg-white rounded-xl shadow-md  w-96 h-full px-12 mt-6 mx-8">
+          <div className="flex justify-center font-bold text-2xl p-6">
+            My Collections
+          </div>
+
+          {collection.map((el) => {
+            return <CollectionListItem number="2" type={el.type} />;
```

Comment:
let the number here actually count the cards depending on the type from the data.js